### ISSUE TITLE
Instance Segmentation for TFLite

### ISSUE THREAD

**Issue by CodeMonkey3435** [tensorflow/tensorflow#52005]
Hello there,
this is not so much of an urgent matter. Nevertheless I wanted to ask, are there any good and maintained open source Instance Segmentation networks that will convert easily to TFLite? I started of with Mask R-CNN but soon realised that the SELECT_OPS flag would slow down the inference time significantly, also because gpu and npu are not available then. I need some model that is natively supported on TFLite (without SELECT_OPS). As far as I understood there are problems with dynamic tensors on GPU/NPU, which is bad because most instance segmenation models use anchors or regions of interest that are resized throughout the model and the segmented individually. I already thought about a single shot detector with sementic segmentation but could not find anything.
I found YOLACT-EDGE (https://github.com/haotian-liu/yolact_edge) until I realized that it is in pytorch :(.

Anybody any ideas?

**Comment by sushreebarsa** [comment]
@CodeMonkey3435 Could you please refer to the [link](https://www.tensorflow.org/lite/examples/segmentation/overview) and let us know if it helps ? Thanks!
